File: admin-panel/src/app/models/Sermon.ts
import { Preacher } from './Preacher';
import { VideoRecording } from './VideoRecording';
export interface SermonDto extends VideoRecording{
  preacherId: number;
}
export interface Sermon extends SermonDto {
  audioPath: string;
  hasVideo: boolean;
  hasAudio: boolean;
  preacherName: string;
  preacher: Preacher;
}

File: lsa-web-apis/Entities/Sermon.cs
using System.Text.Json.Serialization;

namespace lsa_web_apis.Entities;

public class Sermon: VideoRecording
{
    [JsonPropertyName("audioPath")]
    public string AudioPath { get; set; } = "no asignado";

    [JsonPropertyName("preacherId")]
    public int PreacherId { get; set; }
        
    [JsonIgnore]
    public virtual Preacher? Preacher { get; set; }
}
